jerry-main/main-unix.c

```diff
@@ -744,7 +743,7 @@ main (int argc,
     const char *prompt = !no_prompt ? "jerry> " : "";
     bool is_done = false;
 
-    jerry_value_t global_obj_val = jerry_get_global_object ();
+    global_obj_val = jerry_get_global_object ();
```

Comment:
I think this value is initialized to jerry_get_global_object (). Is there no reference counting problem here?